sdk/servicebus/service-bus/README.md

```diff
@@ -378,7 +382,7 @@ If you'd like to contribute to this library, please read the [contributing guide
 [sbclient_tokencred_overload]: https://docs.microsoft.com/javascript/api/@azure/service-bus/servicebusclient?view=azure-node-preview#servicebusclient-string--tokencredential--servicebusclientoptions-
 [sbclient_createsender]: https://docs.microsoft.com/javascript/api/@azure/service-bus/servicebusclient?view=azure-node-preview#createsender-string-
 [sbclient_createreceiver]: https://docs.microsoft.com/javascript/api/@azure/service-bus/servicebusclient?view=azure-node-preview#createreceiver-string--createreceiveroptions--peeklock---
-[sbclient_createsessionreceiver]: https://docs.microsoft.com/javascript/api/@azure/service-bus/servicebusclient?view=azure-node-preview#createsessionreceiver-string--createsessionreceiveroptions--peeklock---
+[sbclient_acceptsession]: https://docs.microsoft.com/javascript/api/@azure/service-bus/servicebusclient?view=azure-node-preview#acceptSession_string__string__AcceptSessionOptions__peekLock___
 [sender]: https://docs.microsoft.com/javascript/api/@azure/service-bus/sender?view=azure-node-preview
 [sender_send]: https://docs.microsoft.com/javascript/api/@azure/service-bus/sender?view=azure-node-preview#sendmessages-servicebusmessage---servicebusmessage-----servicebusmessagebatch--operationoptionsbase-
```

Comment:
This link is bad, it is being redirected to the "latest".

Reply:
I've tried this in two different browser and it's working. I even clicked through the markdown view.

https://github.com/Azure/azure-sdk-for-js/blob/635650cb1510198bee70f0ea677d13778f835532/sdk/servicebus/service-bus/README.md

Try again?

Reply:
To clarify: I wasn't talking about the link in your changes. The link at `sender_send` which is un-edited in the PR is bad.
It takes me to the latest version.
![image](https://user-images.githubusercontent.com/10452642/96626228-13340b80-12c4-11eb-9e37-fedabf628cea.png)

Reply:
Fixed them all.